src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioSymbolNavigationService.cs

```diff
@@ -49,13 +50,14 @@ internal partial class VisualStudioSymbolNavigationService : ForegroundThreadAff
             _metadataAsSourceFileService = componentModel.GetService<IMetadataAsSourceFileService>();
         }
 
-        public bool TryNavigateToSymbol(ISymbol symbol, Project project, CancellationToken cancellationToken, bool usePreviewTab = false)
+        public bool TryNavigateToSymbol(ISymbol symbol, Project project, OptionSet options, CancellationToken cancellationToken)
```

Comment:
It's also called from FAR results too, and the answer is, yes. Call Hierarchy targeted object browser for metadata symbols in Dev12.